server/fileCompress/app.py

```diff
@@ -0,0 +1,95 @@
+import os
+import zipfile
+import json
+import posixpath
+from pprint import pprint
+import boto3
+import shutil
+
+
+def lambda_handler(event, context):
+    # Define the constants
+    TABLE_NAME = "test_table"
+    BUCKET_NAME = "visiontestbucket1337"
+    PRIMARY_KEY = "batch_id"
+    LOOKUP_BATCH_ID = event["queryStringParameters"]["batch_id"]
```

Comment:
set up environment variable for the `table_name` and `bucket_name` in template.yaml. and you can use it here with process.env["VAR_NAME"]

Reply:
Would these be the same environment variables here?
![image](https://user-images.githubusercontent.com/41775764/130604008-381211b2-b814-41c4-a278-fc3cb5664259.png)

Reply:
yes!